Cargo.toml

```diff
@@ -1,8 +1,26 @@
 [profile.release]
 panic = "unwind"
 
+[profile.production]
+inherits = "release"
+
+# Sacrifice compile speed for execution speed by using optimization flags:
+
+# https://doc.rust-lang.org/rustc/linker-plugin-lto.html
+lto = "fat"
+# https://doc.rust-lang.org/rustc/codegen-options/index.html#codegen-units
+codegen-units = 1
+
+[workspace.package]
+version = "2.5.2"
+authors = ["Aventus systems team"]
+homepage = "https://www.aventus.io/"
+repository = "https://github.com/Aventus-Network-Services/avn-node-parachain/"
+edition = "2021"
+rust-version = "1.64.0"
+
 [workspace]
 members = [
     "pallets/*",
-    "primitives/*",
```

Comment:
why remove primitives?

Reply:
We shouldn't, this is a merge conflict. Will restore it